src/transformers/models/deberta/configuration_deberta.py

```diff
@@ -90,7 +90,21 @@ class DebertaConfig(PretrainedConfig):
             `["p2c", "c2p"]`.
         layer_norm_eps (`float`, optional, defaults to 1e-12):
             The epsilon used by the layer normalization layers.
-    """
+
+    Example:
+
+    ```python
+    >>> from transformers import DebertaConfig, DebertaModel
+
+    >>> # Initializing a DeBERTa style configuration
```

Comment:
Let's make it a bit more precise :-)

```python
>>> # Initializing a DeBERTa microsoft/deberta-base style configuration
```

You can check what has been done for `BertConfig`.

Reply:
Done :D